server/paths/prisonApi.ts

```diff
@@ -1,11 +1,11 @@
 import { path } from 'static-path'
 
-const caseloadsPath = path('/api/staff/:staffId/caseloads')
+const caseloadsPath = path('/api/users/me/caseLoads')
```

Comment:
That's not typo in `caseLoads` 😬

Reply:
What an L

Reply:
So the endpoint uses `me`? I think it might still be nicer to use `current`/`currentUser` internally

Reply:
how do you mean? The Prison API endpoint uses "me" (we have no control over this), but internally we're referring to it as `current`